app/models/health/pctp_careplan.rb

```diff
@@ -97,5 +97,30 @@ class PctpCareplan < HealthBase
       where(instrument_id: v1_ids, instrument_type: 'Health::Careplan').
         or(where(instrument_id: v2_ids, instrument_type: 'HealthPctp::Careplan'))
     end
+
+    def expires_on
+      return nil unless instrument.careplan_sent_on.present?
+
+      instrument.careplan_sent_on + 12.months
+    end
+
+    def active?
+      return false unless instrument.careplan_sent_on.present?
+      return false if expired?
+
+      expires_on >= Date.current
+    end
+
+    def expiring?
+      return false unless expires_on.present?
+
+      expires_on > Date.current
```

Comment:
Should there be any date range on this? If I'm reading this correctly it is expiring 12 months after it was sent, but expired... Actually, can you add some comments (and maybe tests for these?)  The logic isn't super clear.

Reply:
I reworked these and added some documentation -- clearer now?

Reply:
Much thank you.  I still think some tests are in order.